src/shared/udev-util.c

```diff
@@ -108,8 +110,31 @@ int udev_parse_config_full(
         return 0;
 }
 
+static int device_new_from_dev_path(const char *devlink, sd_device **ret_device) {
+        struct stat st;
+        int r;
+
+        assert(devlink);
+
+        r = stat(devlink, &st);
+        if (r < 0 && errno == ENOENT)
+                return -ENOENT;
+        if (r < 0)
+                return log_error_errno(errno, "Failed to stat() %s: %m", devlink);
```

Comment:
doesn't matter much, but it might make sense to merge this if brnach and the previous, and always log, but in the ENOENT case do it at debug level. i.e.
```c
if (stat(devlink, &st) < 0) 
        return log_full_errno(errno == ENOENT ? LOG_DEBUG : LOG_ERR, errno: "Failed to stat()"…
```

Reply:
makes sense, done